extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -142,7 +144,9 @@ void FlowFileRepository::initialize_repository() {
   }
   logger_->log_info("Reading existing flow files from database");
 
-  const auto it = opendb->NewIterator(rocksdb::ReadOptions());
+  rocksdb::ReadOptions options;
+  options.verify_checksums = verify_checksums_in_rocksdb_reads_;
+  const auto it = opendb->NewIterator(options);
```

Comment:
```suggestion
  const auto it = opendb->NewIterator(rocksdb::ReadOptions{
    .verify_checksums = verify_checksums_in_rocksdb_reads_
  });
```

Reply:
Unfortunately it does not work with this type:
```
error: designated initializers cannot be used with a non-aggregate type ‘rocksdb::ReadOptions’
```